docker_runner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
+
+if [ "$HOME" = "/home/runner" ]; then
+  my_path="."
+else
+  my_path="/vagrant"
+fi
+
+docker build -t beyond/web_app $my_path
```

Comment:
The build can be performed by GitHub Actions directly. Bash script is an overkill.

Reply:
See above comment where i explain why I choose this approach.